docs/section-2-using-parsers.md

```diff
@@ -651,7 +669,87 @@ And this pattern would match key-value pairs where the `value` is an identifier
 )
 ```
 
-_Note_ - Predicates are not handled directly by the Tree-sitter C library. They are just exposed in a structured form so that higher-level code can perform the filtering. However, higher-level bindings to Tree-sitter like [the Rust crate](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_rust) or the [WebAssembly binding](https://github.com/tree-sitter/tree-sitter/tree/master/lib/binding_web) implement a few common predicates like `#eq?` and `#match?`.
+The prefix "any-" is meant for use with quantified captures. Here's
+an example finding a segment of empty comments
+
+```scheme
+((comment)+ @comment.empty
+  (#any-eq? @comment.empty "//"))
+```
+
+Note that "#any-eq?" will match a quantified capture if
+_any_ of the nodes match the predicate, while by default a quantified capture
+will only match if _all_ the nodes match the predicate.
```

Comment:
While/if you fix the above: Would be useful to also explicitly mention/give an example for the `#any-not-eq?` behavior.